### PR TITLE
CCPhysicsSprite(Box2D/Chipmunk2D) cleanup, some other small improvements...

### PR DESCRIPTION
**Pull request by aismann**
CCPhysicsSpriteBox2D/CCPhysicsSpriteChipmunk2D cleanup
Improve title on :   40:Node::Physics (with Box2D) or (with Chipmunk2D)
Show other window too: 1:ImGuiEXT

### CODE REVIEW

File: extensions/physics-nodes/CCPhysicsSpriteBox2D.h [hunk 1]
```diff
@@ -29,7 +29,6 @@
 #include "extensions/ExtensionExport.h"
 #include "base/CCEventListenerCustom.h"
 
- //#if (CC_ENABLE_CHIPMUNK_INTEGRATION || CC_ENABLE_BOX2D_INTEGRATION)
 
 struct cpBody;
```
Comment (halx99) [inline extensions/physics-nodes/CCPhysicsSpriteBox2D.h:33]: Could be removed?

File: extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h [hunk 2]
```diff
@@ -29,18 +29,14 @@
 #include "extensions/ExtensionExport.h"
 #include "base/CCEventListenerCustom.h"
 
-#if (CC_ENABLE_CHIPMUNK_INTEGRATION || CC_ENABLE_BOX2D_INTEGRATION)
-
 struct cpBody;
 class b2Body;
```
Comment (halx99) [inline extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h:33]: Could be removed?

File: extensions/physics-nodes/CCPhysicsSpriteBox2D.h (hunk 1 above)
Comment (aismann) [reply extensions/physics-nodes/CCPhysicsSpriteBox2D.h:33]: Yes. Forgotten Thanks.

File: extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h (hunk 2 above)
Comment (aismann) [reply extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h:33]: Yes. Forgotten Thanks.

Comment (halx99) [reply extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h:33]: Once you finish cleanup, I will merge

Comment (aismann) [reply extensions/physics-nodes/CCPhysicsSpriteChipmunk2D.h:33]: This PR is the finish cleanup :)
You can merge plz.

**Pull request merged**